stock_unique_order_per_location/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Stock Unique Order Per Location",
+    "summary": "Prevent to mix sales order on same consolidation location.",
```

Comment:
Could be used for any kind of transfer, like internal ones, you have no strict dependency on sale order here.
```suggestion
    "summary": "Prevent to mix transfers on same consolidation location.",
```